tests/_data/snapshots/license-evidence-dev_juice-shop.json.bin

```diff
@@ -5502,15 +5502,6 @@
                 "encoding": "base64"
               }
             }
-          },
-          {
-            "license": {
-              "name": "file: LICENSE.DOCS",
```

Comment:
this is a potential license evidence.

to be honest, i'd rather improve the pattern match in a later ticket/pr, 
than spending too much time with this initial base PR

Reply:
Fine for me.

Reply:
did some research.


this specific one was a false-positive find.
this applied to some docs -- see https://git.coolaj86.com/coolaj86/atob.js/commit/0915ef5d922c086e4b3de4de4380bcb204ac9dcc#diff-8ec9a00bfd09b3190ac6b22251dbb1aa95a0579d

> Docs released under [Creative Commons](https://github.com/node-browser-compat/atob/blob/master/LICENSE.DOCS). 

and these docs were not even shipped: https://www.npmjs.com/package/atob?activeTab=code 


so we should be okay with nit finding this one.

----


for legally binding and completeness of the resulting SBOM: there are no guarantees.  we just do best-effort evidence collection to help others decide and rule.
in the end, that's what layers are for - they assert completeness and do the interpretation. any they might utilize other tools - like [ORT](https://github.com/oss-review-toolkit/ort) - for it.